foundation/eclipselink.core.test/antbuild.xml

```diff
@@ -562,9 +565,10 @@
 
     <!-- Default test target, runs test.class from test.properties (default is LRG). -->
     <target name="test" depends="init-testing">
+        <echo message="is.less.21.jdk: ${is.less.21.jdk}"/>
         <test include="**/junit/**/*Test.java"/>
         <test include="${test.class}"/>
-        <test include="org.eclipse.persistence.testing.tests.security.SecurityTestModel"/>
+        <test include="org.eclipse.persistence.testing.tests.security.SecurityTestModel" if:true="${is.less.21.jdk}"/>
```

Comment:
the `if` here can be avoided by having an extra target with `if`/`unless`

Reply:
OK resolved by recommended way.